spec/factories.rb

```diff
@@ -8,4 +12,9 @@
     sequence(:name) { |n| "name#{n}" }
     sequence(:frequency) { |n| "frequency#{n}" }
   end
+
+  factory :user do
+    sequence(:name) { |n| "user name#{n}" }
```

Comment:
I would only add a sequence for a value that needs to be unique. If it doesn't, I would use straight simple strings with a value that makes sense.

On a different note, frequency, should be set to something like "weekly", or "monthly". Closer you are to the real values, the better.

Reply:
Thanks Elle, I will review these and change.

Reply:
I will do the factory change for frequency when I am doing the two PRs for income controller and expense controller.